invenio_communities/models.py

```diff
@@ -232,9 +248,15 @@ def save_logo(self, stream, filename):
         return False
 
     @classmethod
-    def get(cls, community_id, with_deleted=False):
+    def get(cls, community_id=None, community_name=None, with_deleted=False):
         """Get a community."""
-        q = cls.query.filter_by(id=community_id)
+        assert((community_id is not None) ^ (community_name is not None),
+               'One and only one of community_id or community_name '
```

Comment:
I usually use something like "Provide eitehr X or Y but not both." as error.
It's irrelevant to user of this function that ``community_name`` or ``community_id`` is ``None`` by default and we do some bitwise tricks to get logical XOR on parameters.